src/main/client/src/types/dto/warning.ts

```diff
@@ -0,0 +1,4 @@
+export interface WarningView {
+  message: string
+  type: string
```

Comment:
I think title is more informative. At first I thought this is a type of the alert dialog (info, success, etc.)

Reply:
This is the api's response. We use this interface for the request with axios.

Reply:
lol sad